File: Weekend-Farm-Admin/app/src/main/java/com/farm/weekend/gram/weekend_farm_admin/connect/Connector.java
package com.farm.weekend.gram.weekend_farm_admin.connect;

import okhttp3.OkHttpClient;
import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public final class Connector {
    public static API api;

    static {
        //String TEST_URL = "https://api.github.com/";
        String URL = "URL HERE";
        Retrofit retrofit = new Retrofit.Builder()
                .addConverterFactory(GsonConverterFactory.create())
                .client(new OkHttpClient.Builder().build())
                .baseUrl(URL)
                //.baseUrl(TEST_URL)
                .build();

        api = retrofit.create(API.class);
    }
}

File: Weekend-Farm-Admin/app/src/main/java/com/farm/weekend/gram/weekend_farm_admin/connect/Connector.kt
package com.farm.weekend.gram.weekend_farm_admin.connect

import okhttp3.OkHttpClient
import okhttp3.logging.HttpLoggingInterceptor
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object Connector {

    var api: Api

    private val url = "http://13.124.114.186/admin/"

    init {
        val logger = HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BODY)
        val client = OkHttpClient.Builder().addInterceptor(logger).build()

        val retrofit = Retrofit.Builder()
                .client(client)
                .baseUrl(url)
                .addConverterFactory(GsonConverterFactory.create())
                .build()

        api = retrofit.create(Api::class.java)
    }

}